omod/src/main/resources/config.xml

```diff
@@ -1850,6 +1850,15 @@
 		</description>
 	</globalProperty>
 
+	<!--6275-->
+	<globalProperty>
+		<property>eptsreports.hivCareConceptUuid</property>

```

Comment:
There is a duoplication of this concept... More info about this, on the change requests for the **MEPTS_C1** Pull Request.

Reply:
If C1 is merged, all these will be sorted out